### DISCUSSION TITLE
GitHub Campus Experts Application

### DISCUSSION THREAD

**Discussion by P10swati** [community/community#112316] [GitHub Education]
Hello GitHub Team , I had applied for GitHub Campus Experts Program.The site says I would receive a mail by 12th March regarding my application update. It's 13 March but I haven't received any mail yet. - Please let me know my application updates

**Comment by vmenezesdev** [comment]
I am in the same situation!

**Reply by P10swati** [reply]
They have updated the message on their site. Check it , you'll find a new date !

**Reply by vmenezesdev** [reply]
Thank you!

**Comment by samus-aran** [comment]
Hi there 👋

We’re sorry about the mail not reaching you. There are recent changes to mass email rules that may be affecting the delivery of some messages. However, all applicants can always check their status by logging in at https://education.github.com/campus_experts.
